internal/services/servicebus/servicebus_namespace_disaster_recovery_config_data_source.go

```diff
@@ -94,6 +76,26 @@ func dataSourceServiceBusNamespaceDisasterRecoveryConfig() *pluginsdk.Resource {
 			},
 		},
 	}
+
+	if !features.FivePointOh() {
+		resource.Schema["namespace_id"].AtLeastOneOf = []string{"resource_group_name", "namespace_name", "namespace_id"}
+		resource.Schema["namespace_name"] = &pluginsdk.Schema{
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ValidateFunc: validate.NamespaceName,
+			AtLeastOneOf: []string{"resource_group_name", "namespace_name", "namespace_id"},
+			Deprecated:   "`namespace_name` will be removed in favour of the property `namespace_id` in version 5.0 of the AzureRM Provider.",
+		}
+		resource.Schema["resource_group_name"] = &pluginsdk.Schema{
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ValidateFunc: resourcegroups.ValidateName,
+			AtLeastOneOf: []string{"resource_group_name", "namespace_name", "namespace_id"},
+			Deprecated:   "`resource_group_name` will be removed in favour of the property `namespace_id` in version 5.0 of the AzureRM Provider.",
+		}
```

Comment:
These probably need to have `Computed` set on them, since they will get set by us if they haven't been provided by the user which will cause a diff.

Reply:
I understand. Done.